sda/cmd/api/api_test.go

```diff
@@ -1138,7 +1139,7 @@ func (suite *TestSuite) TestListUserFiles() {
 	testUsers := []string{"user_example.org", "User-B", "User-C"}
 	for _, user := range testUsers {
 		for i := 0; i < 5; i++ {
-			fileID, err := Conf.API.DB.RegisterFile(fmt.Sprintf("/%v/TestGetUserFiles-00%d.c4gh", user, i), user)
+			fileID, err := Conf.API.DB.RegisterFile(fmt.Sprintf("/%v/TestGetUserFiles-00%d.c4gh", user, i), strings.ReplaceAll(user, "_", "@"))
```

Comment:
```suggestion
			fileID, err := Conf.API.DB.RegisterFile(fmt.Sprintf("/%v/TestGetUserFiles-00%d.c4gh", user, i), user)
```
Since now we don't substitute `requester@demo.org` for requester_demo.org, if I understand correctly.

Reply:
Nope, look at what's really happening here and which functions being called.

Reply:
hmm from a look at our databases, I am missing why `UpdateFileEventLog` admits a username with `_` instead of `@` for `upload` events whereas any other event has username with `@`. Not a big deal but I don't get why :sweat_smile:

Reply:
Filepaths needs `_` but the `submission_user` has an `@`. The tests is done backwards since we then only needs to do substitution in one place.